### ISSUE TITLE
the single result lip sync video seems not so good

### ISSUE THREAD

**Issue by Wild-Piggggggg** [soumik-kanad/diff2lip#6]
I got some problem. I tried a single video for lip sync, but the mouth and teeth is not satisfied.

https://github.com/soumik-kanad/diff2lip/assets/72304393/c7266710-29a9-488d-b665-02c49aa6b25f

And I only modified the `inference.sh` and  run it on the terminal by `scripts/inference.sh`↓
<img width="1020" alt="issue2" src="https://github.com/soumik-kanad/diff2lip/assets/72304393/061cfd01-fb15-48e0-9131-adc6adcab536">

**Comment by soumik-kanad** [comment]
Have you tried using the other checkpoint?

**Comment by Wild-Piggggggg** [comment]
Yes. I tried both checkpoints `e7.15_model210000_notUsedInPaper.pt` and `e7.24.1.3_model260000_paper.pt` .  
And here comes another demo.o(╥﹏╥)o

https://github.com/soumik-kanad/diff2lip/assets/72304393/3509de64-3d73-4b64-bb8b-a19bc4d26c13

**Comment by dukesun99** [comment]
Confirm the same problem here. Generated teeth are quite strange.

**Comment by soumik-kanad** [comment]
@Wild-Piggggggg I don't see the flag --is_voxceleb2. Could you please try setting `--is_voxceleb2=False`. 
Also, what is the resolution of this video?

**Comment by chandarren** [comment]
When I start reasoning about the video, it always takes up all my RAM, I've added up to 48G of RAM and the video card isn't being used, why is that?

**Comment by Wild-Piggggggg** [comment]
> @Wild-Piggggggg I don't see the flag --is_voxceleb2. Could you please try setting `--is_voxceleb2=False`. Also, what is the resolution of this video?

Yes, I already set `--is_voxceleb2=False`, and the resolution is `1080×1920`

**Comment by chandarren** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by soumik-kanad** [comment]
> > @Wild-Piggggggg I don't see the flag --is_voxceleb2. Could you please try setting `--is_voxceleb2=False`. Also, what is the resolution of this video?
> 
> Yes, I already set `--is_voxceleb2=False`, and the resolution is `1080×1920`

Can you try at a lower resolution (maybe half) and see if that works?

**Comment by soumik-kanad** [comment]
> When I start reasoning about the video, it always takes up all my RAM, I've added up to 48G of RAM and the video card isn't being used, why is that?

Can you please check where in the code you have the RAM problem? I'm guessing that might be because of face detection. If that is the case, please reduce the face detection batch size.

**Comment by Wild-Piggggggg** [comment]
> > > @Wild-Piggggggg I don't see the flag --is_voxceleb2. Could you please try setting `--is_voxceleb2=False`. Also, what is the resolution of this video?
> > 
> > 
> > Yes, I already set `--is_voxceleb2=False`, and the resolution is `1080×1920`
> 
> Can you try at a lower resolution (maybe half) and see if that works?

I see. I'll try it at a lower resolution.

**Comment by chandarren** [comment]
> > When I start reasoning about the video, it always takes up all my RAM, I've added up to 48G of RAM and the video card isn't being used, why is that?
> 
> Can you please check where in the code you have the RAM problem? I'm guessing that might be because of face detection. If that is the case, please reduce the face detection batch size.

Yes,My config file code reference is from the questioner(Wild-Piggggggg
![Uploading 1123.png…]()
) above and is the same as the code he used

**Comment by chandarren** [comment]
![1123](https://github.com/soumik-kanad/diff2lip/assets/34832078/fa7e281c-62ec-4c21-89d9-78b902e5ab8d)

**Comment by dongyado** [comment]
@Wild-Piggggggg is this problem has been solved by reduce resolutions?

**Comment by rdev2898** [comment]
@delonleo  were you able to solve the issue?

**Comment by chandarren** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
